Btrx 587 int cohorts

LeoForconesi opened the pull request:
## Addresses
[BTRX-587](https://broadinstitute.atlassian.net/browse/BTRX-587): ORSP: Add International Cohorts Questionnaire to NE and NHSR Projects Review and Edit flow

## Changes
- Due to project extra properties already contains a variable named `feeForService` we used `feeForServiceWork` name to differenciate one and another. In order for this change to be updated on  'issue' object, we had to map this variable in the backend

- Created a new component for International Cohorts review so we save boilerplate code in Project and Consent Group

- Some minor tweaks had to be done on `questionnaireWorkflow`. Essentially to differentiate when this questionnaire is being used for creation or edition

- Keep centralizing the use of `isEmpty` method placed in Utils

---

- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Get a thumb from @rushtong
- [ ] **Submitter**: Merge to develop using github's "Squash and Merge" feature
- [ ] **Submitter**: Delete branch after merge

rushtong commented on src/main/webapp/consentGroupReview/ConsentGroupReview.js:
```diff
@@ -579,12 +579,30 @@ class ConsentGroupReview extends Component {
+
+  resetIntCohortsDetermination() {
+    return {
+      projectType: 900,
```
Is there a list of what these numbers correspond to? This would be very confusing to a dev looking at this unless they knew the context (like me ... and I don't know the context). If the values can be constants defined somewhere with descriptive names, that would be more helpful.

LeoForconesi replied:
In order to prevent any confusion, I created an enum(ish) object that contains all possible determination types outputs coming from questionnaire workflow implementation. So now they are all together in one place. 

On the other hand, it wans't neccesary to inicialize any type of determination for this object, you will see them null on constructors and for reset purpouses.

rushtong replied:
👍

LeoForconesi commented:
@rushtong i Corrected the code, please check again. 
Also added a validation scenario for Legacy projects. In these cases International Cohorts and Info Security shouldn't be required to submit an edit/suggestion
